src/ray/object_manager/pull_manager.h

```diff
@@ -172,15 +177,16 @@ class PullManager {
 
   /// Activate the next pull request in the queue. This will start pulls for
   /// any objects in the request that are not already being pulled.
-  bool ActivateNextPullBundleRequest(
-      const std::map<uint64_t, PullBundleRequest>::iterator &next_request_it,
-      std::vector<ObjectID> *objects_to_pull);
+  bool ActivateNextPullBundleRequest(const Queue &bundles,
```

Comment:
Can you explain the params & return value's meaning in the docstring?

Reply:
I don't think this is usually necessary for helper functions, but I will add a note about the return value.

Reply:
Yeah explaining return value sounds enough to me!